src/shared/lib/validation/user.ts

```diff
@@ -1,5 +1,7 @@
+import { uniq } from 'lodash';
+import SKILLS from 'shared/lib/data/skills';
```

Comment:
See comment from SWU opp PR — this module should be duplicated and named as `capabilities`